eth2/utils/tree_hash/src/merkleize_padded.rs

```diff
@@ -0,0 +1,354 @@
+use super::BYTES_PER_CHUNK;
+use hashing::hash;
+
+/// The size of the cache that stores padding nodes for a given height.
+///
+/// Currently, we panic if we encounter a tree with a height larger than `MAX_TREE_DEPTH`.
+pub const MAX_TREE_DEPTH: usize = 32;
```

Comment:
In light of the validator list being `2^40` entries, perhaps we should make this a bit higher, maybe 48? We don't seem to exponentiate with it, so even 64 could be OK.

Reply:
Good idea!

Reply:
Set to 64 :)